appservice/src/main/java/nu/yona/server/subscriptions/rest/UserSignInConfirmationDTO.java

```diff
@@ -0,0 +1,22 @@
+package nu.yona.server.subscriptions.rest;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+@JsonRootName("userSignInConfirmation")
+public class UserSignInConfirmationDTO
```

Comment:
Rename into ``MobileNumberConfirmationDTO`` (don't forget the ``JsonRootName``).